arcgis-ios-sdk-samples/Layers/Display WFS/README.md

```diff
@@ -0,0 +1,33 @@
+# Display a WFS layer
+
+This sample demonstrates how to display a layer from a WFS service, requesting only features for the current extent.
+
+WFS is an open standard with functionality similar to ArcGIS feature services. Runtime support for WFS allows you to interoperate with open systems, which are often used in inter-agency efforts, like those for disaster relief.
+
+![](image1.png)
+
+## How to use the sample
+
+Pan and zoom to see features within the current map extent.
+
+## How it works
```

Comment:
Please ensure that each header has an empty line between it and its contents. I've seen a case where a list didn't render properly on iOS when there wasn't an empty line.